### ISSUE TITLE
Inter2p for point cloud data

### ISSUE THREAD

**Issue by DominikFle** [f0uriest/interpax#42]
Hello there,
Great library!
Am i right, that your implementation of interp2 supports structured grids, but not scattered points data ?
So to compare it to the old scipy version of [scipy interp2d](https://docs.scipy.org/doc/scipy-1.10.1/reference/generated/scipy.interpolate.interp2d.html)
you implemented not the entire functionality right ?
I am looking for a way to interpolate scattered data, like in scipy interp2d, or now the [LinearNDInterpolator ](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.LinearNDInterpolator.html).
Thank you for clarifying!

**Comment by f0uriest** [comment]
That's correct. Splines for unstructured data is a lot trickier and usually needs some sort of tessellation which I haven't found a good implementation of in Jax, so it's not something on the roadmap, but if you have any ideas I'd welcome contributions.
